diff --git a/index.js b/index.js
index d45e67d91d38c91d05e68e8b338e4bc2de591dcc..0679d6087914857bac42ddd3261c664cbc8fa745 100644
--- a/index.js
+++ b/index.js
@@ -1,5 +1,6 @@
 'use strict';
 const strictUriEncode = require('strict-uri-encode');
-const decodeComponent = require('decode-uri-component');
+const decodeUriComponent = require('decode-uri-component');
+const decodeComponent = decodeUriComponent.default || decodeUriComponent;
 const splitOnFirst = require('split-on-first');
 const filterObject = require('filter-obj');